recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.0" %}
+{% set git_branch = "master" %}
+{% set build_number = "0" %}
+
+package:
+  name: ale
+  version: {{ version }}
+
+source:
+  git_url: 'https://github.com/USGS-Astrogeology/ale.git'
+  git_branch: {{ git_branch }}
+
+build:
+  number: {{ build_number }}
+
+# Shotgun strat on requirements until we can narrow them down
+requirements:
+  build:
+  - python
+  - cmake>=3.10
+  - gsl
+  - openblas>=0.3.0
+  - eigen
+  - spiceypy
+  - numpy
+  - python-dateutil
+  - pvl
+  run:
+  - python
+  - cmake>=3.10
+  - gsl
+  - openblas>=0.3.0
+  - eigen
+  - spiceypy
+  - numpy
+  - python-dateutil
+  - pvl
+
+# Add the tests eventually
+# test:
+#
+
+# about:
+#   home: https://developers.google.com/protocol-buffers/
+#   license: BSD 3-Clause
```

Comment:
What license do we want to use for ale?

Reply:
Yeah, I left an example "about" section commented out for reference, but forgot to ask about specific ale implementation... Any thoughts?

Reply:
Yeah! This is less a comment on your actual PR and more a question that should have gone in a ticket. I'm not sure.

Reply:
I think we can't use a license? My understandings is that we have to push everything we do into the public domain. @jlaura has looked into this more.

Reply:
@jessemapel Isn't pushing something into the public domain still the sort of thing that requires a license to indicate that it is all in the public domain?